Model/Config.php

```diff
@@ -186,6 +186,33 @@ private function getJwt(string $scopeType = ScopeInterface::SCOPE_STORE, string
         return $this->jwt;
     }
 
+    /**
+     *
+     * @param string $config
+     * @return string
+     */
+    public function getPaypalBlockStyling(string $config): string
```

Comment:
Is it possible to add a tests for this method to ensure the correct styling is returned for the different types of config?